test/test_sparse.py

```diff
@@ -107,9 +75,17 @@ def randn(self, *args, **kwargs):
         Variant of torch.randn that also works in the TEST_CUDA case.
         """
         # TODO: Put this in torch.cuda.randn
-        return self.value_empty(*args, **kwargs).normal_()
+        return torch.empty(*args, **kwargs).normal_()
+
+    @dtypes(torch.double)
```

Comment:
Prefer torch.float if using a single floating point type (it's OK to not change this in this PR if it would be a pain)

Reply:
I was trying to preserve the same test behavior, in previous test_sparse, the value_dtype is torch.float64;
https://github.com/pytorch/pytorch/blob/0e03a2978ad5ed2be09f3e4691b2d04d6d4c6c90/test/test_sparse.py#L60 

However I can change that, I understand that torch.float is prefered because it will make tests lighter